### ISSUE TITLE
Can we specify survey_id?

### ISSUE THREAD

**Issue by ttkrpink** [sdaps/sdaps#262]
I have some scans already. When I am trying to create a new project for the old scans, the recognize and reorder will through out wrong survey_id warning. 

I realizd that the survey id is generated randomly at the time of setup. Is there a way to specify survey_id in Latex or command line to get rid of the warning?

Thanks.

**Comment by benzea** [comment]
Right, the survey ID is stored in the sqlite database. So, not that nicely, you might even need to modify the contained json string. Or, patch model/survey.py to modify the load function and override the value. Do any operation that requires saving and it should be modified.

**Comment by ttkrpink** [comment]
So, can I ignore all the warnings that  survey_id does not match in the reorder and recognize output?
